src/components/steps/org-units/OrgUnitsStep.tsx

```diff
@@ -53,6 +46,7 @@ const OrgUnitsStep: React.FC<StepProps> = ({ api, project, onChange }) => {
                     selected={project.organisationUnits.map(ou => ou.path)}
                     controls={controls}
                     rootIds={rootIds}
+                    levels={[1, 2, 3]}
```

Comment:
@laurapareja  I think this is not enough to make this PR https://github.com/EyeSeeTea/d2-ui-components/pull/89 works right? I think this is some old code you have just forgotten. Although it is not the best practice I am fine with creating a PR that contains the user roles and modifications for the organisation unit. Can you please double check if you have forgotten pushing your changes?

Reply:
Tomorrow I'll remove this old code. Related to ou-level there is a branch where I was checking those changes: 
https://github.com/EyeSeeTea/project-monitoring-app/compare/feature/ou-level-wizard
I can do tomorrow a PR to link d2 to this project, so everything is linked.